apps/pie-docs/src/components/modal/overview/overview.md

```diff
@@ -0,0 +1,424 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: Modal
+    order: 1
+eleventyComputed:
+    sizes: "{% include './sizes.json' %}"
+shouldShowContents: true,
+---
+
+
+## Overview
+The purpose of modals is to focus the user's attention on a specific task, message, or interaction, and to prevent interaction with other elements on the page while the modal is active.
+
+Modals are commonly used for tasks such as displaying notifications, presenting detailed information, requesting user input, or confirming critical actions.
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/overview.svg",
+    alt: "A modal containing a heading, body copy and a button placed on the bottom right corner."
+} %}
+
+---
+
+## Dos and Don’ts
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+            "Use to inform users about a task can can contain information, require decisions or involve multiple tasks."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Modal’s interrupt a user’s flow by design, so while effective when used correctly; Modal’s should be used sparingly to limit disruption to the user."
+        ]
+    }
+} %}
+
+
+---
+
+## Anatomy
+
+{% contentPageImage {
+    src: "../../../assets/img/components/modal/overview/anatomy.png",
+    alt: "Anatomy of a modal.",
+    shouldShowPadding: true
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Back (optional)**: The chevron Icon button allows the user to return back a stage in a multi-step modal.",
+        "**Title**: Gives the users an overview of the content.",
+        "**Close**: The close Icon button will close the modal without submitting any data.",
+        "**Main content**: Open slot for any content required.",
+        "**CTAs (optional)**: Single or dual call to action Buttons to outline the user’s next options.",
+        "**Overlay**: Screen overlay that obscures the on-page content (1.Elements/1. Backgrounds/3. Page/Overlay)."
```

Comment:
What does this bit in brackets mean? It's not immediately obvious to me

Reply:
I'm not actually sure, @nazanin 
It's for this image
<img width="800" alt="Screenshot 2023-11-24 at 11 56 12" src="https://github.com/justeattakeaway/pie/assets/24740015/7c4d7ac9-7a30-4bd2-ad95-bc0468f62936">

Reply:
@xander-marjoram @FayeCarter That's the address where the token exist on design, but I actually have no idea why that piece of information exist, there was a rush to get this up and no optimization has been done on the content of modal's doc, so any recommendations and rewordings are welcome, you can delete it now, I'll track to see why it's there and an alternative

Reply:
👍  thanks